### DISCUSSION TITLE
Why does implicit OSCAL namespace lack a version number?

### DISCUSSION THREAD

**Discussion by galtm** [usnistgov/OSCAL#1258] [Q&A] [answered]
Related to #1247 : Why is the implicitly assumed OSCAL namespace `http://csrc.nist.gov/ns/oscal` with no version number, when the OSCAL elements are in namespace `http://csrc.nist.gov/ns/oscal/1.0` with a version number?

**Comment by david-waltermire** [comment, marked as answer]
We want things like a `prop` or `part` name to persist across multiple major OSCAL revisions. This is why the namespace used in these contexts is not major version specific.

The XML namespace is intended to be major version specific, since major version updates will allow backwards compatibility breaking changes. Yes, `1.0` includes a minor version. We will fix this when the first `2.x` release is made.

This is why the part/prop namespace differs from the XML namespace.

**Reply by galtm** [reply]
Thanks for your explanation, @david-waltermire-nist.
